models/tiegcm/model_mod.f90

```diff
@@ -251,143 +222,36 @@ subroutine static_init_model()
    write(logfileunit,*)'static_init_model: debug level is ',debug
 endif
 
-! Read in TIEGCM namelist input file (just for definition)
-! Read in TIEGCM grid definition etc from TIEGCM restart file
-! Read in TIEGCM auxiliary variables from TIEGCM 'secondary' file
-
-call read_TIEGCM_namelist(tiegcm_namelist_file_name)
+! Read in TIEGCM grid definition TIEGCM restart file
 call read_TIEGCM_definition(tiegcm_restart_file_name)
-call read_TIEGCM_secondary(tiegcm_secondary_file_name)
-
-! error-check and convert namelist input to variable_table
-! and the 'progvar' database in the scope of the module
-call verify_variables(variables, nfields)
 
-! Compute overall model size
-
-model_size = progvar(nfields)%indexN
-
-if (do_output() .and. (debug > 0)) then
-   write(*,*) 'nlon  = ', nlon
-   write(*,*) 'nlat  = ', nlat
-   write(*,*) 'nlev  = ', nlev
-   write(*,*) 'nilev = ', nilev
-   write(*,*) 'model_size = ', model_size
-   if (estimate_f10_7) &
-   write(*,*) 'estimating f10.7 ... init value ',f10_7
+if ( estimate_f10_7 ) then
+   call error_handler(E_MSG, 'f10_7 part of DART state', source)
 endif
 
-allocate (ens_mean(model_size))
-ens_mean = MISSING_R8
+! error-check, convert namelist input to variable_table, and build the
+! state structure
+call verify_variables()
 
-! Might as well use the Gregorian Calendar
 call set_calendar_type('Gregorian')
 
-! Convert the last year/doy/hour/minute to a dart time.
-state_time = get_state_time(tiegcm_restart_file_name)
+! Convert the last year/day/hour/minute to a dart time.
```

Comment:
does tiegcm time use 'doy' == day of year, 1-366?  if so, this isn't really a typo.  but i suppose day is just as clear, to be honest.

Reply:
oh, I read doy and day as the same word. 

tiegcm uses day,hour,minute. The tiegcm user guide uses 'day' for day-of-year, e.g.

```
STOP integer(3) Model stop time (day,hour,minute)
````

so it makes sense for DART to follow that.